toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -423,6 +423,8 @@ func downloadSingleDeltaRPM(realDependencyGraph *pkggraph.PkgGraph, buildNode *p
 			logger.Log.Warnf("Can't find delta RPM to download for %s: %s (local copy may be newer than published version)", fullyQualifiedRpmName, err)
 			return nil
 		}
+	} else {
```

Comment:
This seems unrelated to the PR, and probably deserves its own PR. I am thinking from the perspective where we might want to roll-back in some imaginary scenario, and would end up erroneously deleting this edge case in graphpkgfetcher.

Reply:
This doesn't materially affect the behavior of the tool, so I don't mind bundling it (and it was the debugging information that allowed me to diagnose the issue this is solving so I feel its at least partially related).